src/rt/sched/cown.h

```diff
@@ -1363,6 +1370,11 @@ namespace verona::rt
         MultiMessage::make_body(alloc, count, cowns, &unmute_behaviour);
       return MultiMessage::make_message(alloc, body, epoch);
     }
+
+    void send_remove_poller_msg(Cown* c)
+    {
+      schedule_lambda([=]() { Scheduler::local()->poller_remove(c); });
```

Comment:
Shouldn't the behaviour be scheduled on the token cown (`this`)?
```suggestion
      schedule_lambda(this, [=]() { Scheduler::local()->poller_remove(c); });
```